zulipterminal/helper.py

```diff
@@ -657,6 +657,19 @@ def hash_util_decode(string: str) -> str:
     return unquote(string.replace('.', '%'))
 
 
+def get_unused_fence(content: Any) -> Any:
```

Comment:
You don't have types specified here - can we improve this?

Does this come from a translation of a function in the server? We should reference it as a comment, if so, and perhaps when. Also if you used some particular logic to translate it.

Reply:
I have improved type annotations to `str`.
Not necessarily, its referred from `zulip/static/shared/js/fenced_code.js`, if I am correct in assuming by server you mean `zulip/zerver`. I have added short comments addressing this function.